CITATION.cff

```diff
@@ -0,0 +1,30 @@
+# YAML 1.2
+---
+title: weldx
+version:
+date-released:
+authors:
+  -
+    affiliation: "Bundesanstalt für Materialforschung und -prüfung (BAM)"
+    family-names: Fabry
+    given-names: Cagtay
+    orcid: "https://orcid.org/0000-0002-0788-8079"
+  -
+    affiliation: "Bundesanstalt für Materialforschung und -prüfung (BAM)"
+    family-names: Hirthammer
+    given-names: Volker
+    orcid: " https://orcid.org/0000-0002-0160-969X"
+  -
+    affiliation: "Bundesanstalt für Materialforschung und -prüfung (BAM)"
+    family-names: Scherer
+    given-names: Martin K.
+    orcid: "https://orcid.org/0000-0002-7983-4387"
+  -
+    affiliation: "Bundesanstalt für Materialforschung und -prüfung (BAM)"
+    family-names: Pittner
+    given-names: Andreas
+    orcid: "https://orcid.org/0000-0002-5019-7775"
+cff-version: "1.1.0"
+message: "If you use this software, please cite it as below."
+repository-code: "https://github.com/BAMWelDX/weldx"
+...
```

Comment:
should this not point to an article instead of the repo?

Reply:
yes should be updated to point to the latest publication once available